inst/test-data/occ/occ-data.R

```diff
@@ -0,0 +1,93 @@
+
+#'
+#' # Examples
+#'
+library(dplyr)
+library(mrgsolve)
+library(lastdose)
+library(here)
+
+csv <- function(x, file) {
+  file <- here("inst/data/occ/", file)
```

Comment:
This should now be `inst/test-data/occ/` to account for the rename in 4d05903 (rename data directory, 2025-01-02).